SimPEG/regularization/pgi.py

```diff
@@ -108,6 +261,15 @@ def set_weights(self, **weights):
 
     @property
     def gmm(self):
+        """Gaussian mixture model.
+
+        Returns
+        -------
+        None, SimPEG.utils.WeightedGaussianMixture
```

Comment:
I think this property is never `None`. In case `gmm` was passed as `None` in the `__init__`, the `self._gmm` attribute will be set here as a copy of the `gmmref`.
```suggestion
        SimPEG.utils.WeightedGaussianMixture
```